jetstream/jetstream.go

```diff
@@ -183,6 +183,12 @@ type (
 		// DeleteConsumer removes a consumer with given name from a stream.
 		// If consumer does not exist, ErrConsumerNotFound is returned.
 		DeleteConsumer(ctx context.Context, stream string, consumer string) error
+
+		// PauseConsumer pauses a stream. If pauseUntil is provided, the stream will be paused until that time.
```

Comment:
If you want to pause a consumer, you have to provide `pauseUntil`, so the comment is a bit misleading. Also, I would change the signature to accept value, not a pointer.

Also, PauseConsumer pauses a **consumer**, not a stream